### ISSUE TITLE
TableRow Clicked property?

### ISSUE THREAD

**Issue by aford22** [stsrki/Blazorise#64]
Hello,

I am using your TableContainer component like this:

```
        <TableContainer IsHoverable="true">
            <TableHeader>
                <TableRow>
                    <TableHeaderCell>Id</TableHeaderCell>
                    <TableHeaderCell>TimeStamp</TableHeaderCell>
                </TableRow>
            </TableHeader>
            <TableBody>
                @foreach (var e in AllErrorEntries)
                {
                    <TableRow Clicked="@HandleUseCaseItemClick">
                        <TableRowCell>@e.Id</TableRowCell>
                        <TableRowCell>@e.TimeStamp</TableRowCell>
                    </TableRow>
                }
            </TableBody>
        </TableContainer>
```

However the browser generates the error:
Object of type 'Blazorise.TableRowCell' does not have a property matching the name 'Clicked'

Couple of questions:
1) How can I tell from your source code what the valid properties are? Trying to help myself without bothering you every time here :-) The documentation at https://blazorise.com/docs/ does not tell me what the properties of the Table component are...
2) How do you intend people to capture and react to clicks on your Table component? What should the code be?

**Comment by stsrki** [comment]
You're getting error because there's no Clicked event parameter implemented on TableRowCell.

1. Documentation is still not done and also there is no doc for table component :( Maybe I should create new page for table...
2. I can add Clicked event to the components you need. I think it should be pretty easy to do. Just give me a list of components that you need it for.

PS. Please don't be afraid to bother me. There is so much things to think of and so much jet to do that I just forget to add them. Also Blazorise is created for users like you that can give me insight how to design components and what is still missing.

**Comment by stsrki** [comment]
PPS. How fast you need this feature? I have started work on 0.7 and it will take some time. If you need it fast I can do it in a quick 0.6.4

**Comment by aford22** [comment]
Hey there. Clicked events on the following components would be just awesome:
TableRow
TableHeaderCell
TableRowCell

I am trying to build out my UI but without a way to interact with the table its impossible to implement things like sorting or master/detail... So yes, I would appreciate these in 0.6.4. I can test them as soon as you are ready and provide quick feedback too!!

**Comment by stsrki** [comment]
If I understand your `myParam` it is some kind of user-defined data? Is it OK for this attribute to be "object" type?

**Comment by stsrki** [comment]
@aford22 What about other mouse events? onmousemove, onmouseover, onmousewheel etc.

**Comment by aford22** [comment]
I have no particular need for those functions, no.

**Comment by stsrki** [comment]
Just released v0.6.4. Please try it and let me know if t's OK.

**Comment by aford22** [comment]
TableRow works fine. Going to test the other two over the weekend.

Cheers mate.
